docs/Install-with-Docker.md

```diff
@@ -100,6 +109,35 @@ SELECT * FROM realmlist;
 UPDATE realmlist SET address='<SERVER PUBLIC IP ADDRESS>';
 ```
 
+### How to run the worlserver with GDB
+
+1. Create a `config.sh` file under the `/conf/` directory of the azerothcore-wotlk repository
+2. Add this configuration inside: `AC_RESTARTER_WITHGDB=true`. It will configure the restarter used by our docker services to use GDB instead of the binaries directly
+
+### How to use the dev-container
```

Comment:
what about a short introduction about what the dev-container actually is and how it can be useful when developing?